docs/_docs/01-ug-quick-start-guide.md

```diff
@@ -59,7 +59,16 @@ Start the Spark Scala shell as follows:
 ```
 
 #### PySpark
-Support for Pyspark is [on the way](https://github.com/microsoft/hyperspace/pull/36).
+Install Pyspark by running the following:
+
+```
+pip install pyspark==2.4.2
+```
+
+Then, run PySpark with the Hyperspace package:
+```
+pyspark --packages com.microsoft.hyperspace:hyperspace-core_2.11:0.1.0
```

Comment:
0.1.0 wouldn't work no?

Reply:
yea right. need to be the new version to be released.